src/test/end-to-end/tests/custom-steps.ts

```diff
@@ -171,5 +172,12 @@ export = function () {
             this.click('Continue');
             this.waitForText('User Deleted');
         },
+        checkA11y(fileName: string) {
+            this.runA11yCheck({ reportFileName: fileName });
+            this.usePlaywrightTo('Run accessibility tests', async ({ page }) => {
+                await injectAxe(page);
+                await checkA11y(page);
```

Comment:
In terms of the question the other day regarding logging but not failing the build (just while we are in the process of this initial setup / fix phase) - it looks like there's a 'skipFailures' option for the checkA11y method: https://github.com/abhinaba-ghosh/axe-playwright.

It will stick log the failures so we can review them, but not fail the test.

Note we'll need to add a ticket to remove this once all of the tests are set up / failures fixed.